src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/base_js.html.twig

```diff
@@ -338,13 +338,14 @@
                     var tabNavigation = document.createElement('ul');
                     tabNavigation.className = 'tab-navigation';
 
+                    var selectedTabId = 'tab-' + i + '-0'; /* select the first tab by default */
```

Comment:
does it twigbundle need the same fix? see comment in 3.4 

https://github.com/symfony/symfony/blob/cfb243e61a2ecdc9a5eb26aaff8897ca74bfad8d/src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/base_js.html.twig#L1-L2

also the reason im a bit skeptical about 2.8... it's new behavior and things have changed as of 3.4. Not sure if the merge will be conflicting.. but tend to prefer 3.4 somehow :)